bolt/include/bolt/Core/BinaryFunction.h

```diff
@@ -75,6 +75,13 @@ enum IndirectCallPromotionType : char {
   ICP_ALL          /// Perform ICP on calls and jump tables.
 };
 
+/// Hash functions supported for BF/BB hashing.
+enum class HashFunction : char {
+  StdHash, /// std::hash, implementation is platform-dependent. Provided for
+           /// backwards compatibility.
+  Xxh3,    /// llvm::xxh3_64bits, the default.
```

Comment:
Tiny nit: `s/Xxh3/XXH3/`. If the name is all caps, keep it all caps or lower all letters. Not sure if such rule is listed in LLVM docs.